package.json

```diff
@@ -34,6 +34,7 @@
   "dependencies": {
     "aws-sdk": "^2.94.0",
     "chai": "^4.1.2",
+    "global": "^4.3.2",
```

Comment:
Is this dependency used?

Reply:
https://www.npmjs.com/package/global
Should be removed.

Reply:
yes not required. i will remove